doc/config_manager.rst

```diff
@@ -71,11 +71,12 @@ Examples
 ``dnf config-manager --dump``
     Display main DNF configuration.
 
-``dnf config-manager repo --dump``
-    Display configuration of repo.
+``dnf config-manager <repoid> --dump``
+    Display configuration of repository identified by <repoid>.
```

Comment:
I'm reasonably sure there should be a particle to be grammatically correct. It's probably the indefinite one (i.e. _... configuration of **a** repository ..._), my previous suggestion was probably wrong... I wish we had a native speaker to handle this for us...

(same goes for the hunk below)